tests/serac_profiling.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include <exception>
+
+#include <gtest/gtest.h>
+
+#include "serac/infrastructure/cli.hpp"
+#include "serac/infrastructure/initialize.hpp"
+#include "serac/infrastructure/profiling.hpp"
+#include "serac/numerics/mesh_utils.hpp"
+#include <cstring>
+
+namespace serac {
+  
+TEST(serac_profiling, mesh_refinement)
```

Comment:
Since we're not using gtest anywhere in this, it might be better to have this be an example?

Reply:
I added a simple smoke check on the number of elements after refinement.